timesheetService/TimesheetAggregate.js

```diff
@@ -45,12 +46,12 @@ class Timesheet extends AggregateBase {
   create(createTimesheetPayload) {
     if (this.version != SEED_VERSION)
       throw new InvalidOperationException("Create can only be executed as the first action.");
-    var timesheetCreatedEvent = {
+    let timesheetCreatedEvent = {
       "id" : this.id,
       "eventId" : 1,
       "eventType" : "TimesheetCreated",
-      "sourceLambdaEvent" : JSON.stringify(createTimesheetPayload),
-      "event" : createTimesheetPayload.body
+      "sourceLambdaEvent" : createTimesheetPayload,
+      "event" : JSON.parse(createTimesheetPayload.body) 
```

Comment:
The nested event should be valid JSON, the fact that the db client will re serilaise it to a string is not relevant to me - the event should be valid at the time of raising it. The stringifying it is just a limitation of DynamoDB